.github/workflows/spelling.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       - name: Run check
         uses: codespell-project/actions-codespell@master
         with:
-          skip: ./src/control/vendor,./.git
+          skip: ./src/control/vendor,./.git,./src/rdb/raft
```

Comment:
src/rdb/raft is a git submodule which shouldn't be checked out so shouldn't need excluding.

Reply:
When I run it locally, it was checked. Do not know that it was not checked with GitHubActions. I am going to remove it.

- [ ] Remove useless skip item